eos_data_distribution/avahi_monitor.py

```diff
@@ -188,6 +192,12 @@ def remove_nexthop(self, faceUri):
         return check_call(["nfdc", "remove-nexthop",
                            str(SUBSCRIPTIONS_SOMA), faceUri])
 
+    def start_edge(self):
+        return check_call(["systemctl", "start", "edd-soma-subscriptions-producer"])
```

Comment:
What Avahi-less machines? If `edd-soma-subscriptions-producer` had a config file allowing you to set the policy for when it would advertise SOMA subscriptions over NDN, the file could allow you to (for example) choose between `always`, `never` and `mdns-gateway` which would enable it based on the value of `self._routed` from this code.